netci.groovy

```diff
@@ -14,17 +14,10 @@ def windowsUnitTestMachine = 'win2016-base'
 
 static void addRoslynJob(def myJob, String jobName, String branchName, Boolean isPr, String triggerPhraseExtra, Boolean triggerPhraseOnly = false) {
   def archiveSettings = new ArchivalSettings()
-  archiveSettings.addFiles('Binaries/**/*.pdb')
-  archiveSettings.addFiles('Binaries/**/*.xml')
-  archiveSettings.addFiles('Binaries/**/*.log')
```

Comment:
Probably want to keep this line, so we have both the screenshot and the exception that accompanies it.

Reply:
@jaredpar this wasn't addressed.

Reply:
The log file?